ifrs17/Import/4ImportScope-TechnicalMargin.ipynb

```diff
@@ -338,7 +338,9 @@
                 "\n    static ApplicabilityBuilder ScopeApplicabilityBuilder(ApplicabilityBuilder builder) =>",
                 "\n        builder.ForScope<TechnicalMarginForAM>(s => s.WithApplicability<TechnicalMarginForAmForPaa>(x => x.Identity.ValuationApproach == ValuationApproaches.PAA));   ",
                 "\n ",
-                "\n    double TechnicalMargin.Value => -1d * AggregatedValue * GetScope<CurrentPeriodAmortizationFactor>((Identity, AmountTypes.CU, 0), o => o.WithContext(EconomicBasis)).Value;",
+                "\n    double TechnicalMargin.Value => Math.Abs(AggregatedValue) > Precision",
+                "\n        ? -1d * AggregatedValue * GetScope<CurrentPeriodAmortizationFactor>((Identity, AmountTypes.CU, 0), o => o.WithContext(EconomicBasis)).Value",
```

Comment:
You have another style for the ternary operator here, in multiple lines, when compared to the ones above.